.github/workflows/ci.yml

```
@@ -30,18 +30,7 @@ jobs:
       matrix:
         os: [ubuntu-latest]
         eslint: [6, 7, 8]
-        node: [8.10.0, 8, 10, 12, 14, 16]
-        exclude:
-          - eslint: 8
-            node: 10
-          - eslint: 8
-            node: 8
-          - eslint: 8
-            node: 8.10.0
-          - eslint: 7
-            node: 8
-          - eslint: 7
-            node: 8.10.0
+        node: [12, 14, 16]
```

To ensure it still works against the absolute lowest supported version:

```suggestion
        node: [12.22.0, 12, 14, 16]
```

(Previously we've discussed testing against the lowest supported patch of _each_ major Node version. We decided that we'd only add e.g. `16.0.0` in addition to `16` if we accidentally broke `16.0.0` but didn't catch it because it still passed in `16`.)

---

done